src/components/ResultCard/ResultCard.js

```diff
@@ -4,13 +4,26 @@ import { Card, CardBody } from 'reactstrap';
 
 import './ResultCard.scss';
 
-export function ResultCard({ body, isSuccessful = true }) {
+export function ResultCard({ body, status = 'success' }) {
```

Comment:
Its a breaking change regarding the ResultCard (the prop changed), the change needs to be ensured everywhere where it is used